src/layers/src/Pane.js

```diff
@@ -178,11 +181,17 @@ class Pane extends PureComponent {
         borderLeft={_borderLeft}
         boxShadow={elevationStyle}
         background={theme.getBackground(background)}
-        css={{
-          ...css,
-          ...hoverElevationStyle,
-          ...activeElevationStyle
-        }}
+        className={cx(
+          className,
+          glamorCss(
+            {
+              ...css,
+              ...hoverElevationStyle,
+              ...activeElevationStyle
+            },
+            className
```

Comment:
Why include `className` here?

Reply:
Good catch, that was left over from some experimenting I was doing at the beginning.